install_config/persistent_storage/persistent_storage_csi.adoc

```diff
@@ -0,0 +1,408 @@
+
+[NOTE]
+====
+`attach` and `detach` operations typically require the CSI driver to use credentials to the storage backend. It is recommended to run the external attacher pod on xref:../../install_config/install/advanced_install.adoc#configuring-dedicated-infrastructure-nodes[infrastructure nodes] so the credentials never leak to user processes, even in the event of a catastrophic security breach on a node.
+====
+
+[NOTE]
+====
+The external attacher must also run for CSI drivers that don not support third-party attach/detach operations. External attacher will not issue any `ControllerPublish` or `ControllerUnpublish` operations to CSI driver, however, it still must run to implement the necessary {product-title} attachment API.
+====
+
+=== External CSI Provisioner Deployment
+
+External CSI provisioner is a deployment that deploys single or several pods with two containers:
+
+  * External CSI provisioner container itself that translates `provision` and `delete` calls from {product-title} to respective `CreateVolume` and `DeleteVolume` calls to CSI driver.
+  * CSI driver container.
+
+The CSI provisioner container talks to the CSI driver container using Unix Domain Sockets, ensuring that no CSI communication leaves the pod. The CSI driver is not accessible from outside of the pod.
+
+Deployment can be used to scale the number of provisioners up or down.
+
+[NOTE]
+====
+`provision` and `delete` operations typically require the CSI driver to use credentials to the storage backend. It is recommended to run the external attacher pod on xref:../../install_config/install/advanced_install.adoc#configuring-dedicated-infrastructure-nodes[infrastructure nodes] so that the credentials never leak to the user processes, even in the event of a catastrophic security breach on a node.
+====
+
+=== CSI Driver DaemonSet
+
+Finally, the CSI driver DaemonSet runs a pod on every node that allows {product-title} to mount storage provided by CSI driver to the node and use it in user workloads (pods) as persistent volumes (PVs). The pod with the CSI driver installed contains the following containers:
+
+* CSI driver registrar, which registers the CSI driver into the `openshift-node` service running on the node. The `openshift-node` process running on the node then directly talks to the CSI driver using the Unix Domain Socket available on the node.
+* CSI driver.
+
+The CSI driver deployed on the node should have as few credentials to the storage backend as possible. {product-title} will only use the node plugin set of CSI calls such as `NodePublish`/`NodeUnpublish` and `NodeStage`/`NodeUnstage` (if implemented).
+
+== Example Deployment
+
+Since {product-title} does not ship with any CSI driver installed, this example shows how to deploy a community driver for OpenStack Cinder in {product-title}.
+
+1. Create a new project where the CSI components will run and a new service account that will run the components
```

Comment:
We should use the . format for numbering and the + to keep the example inline with the text

components.